src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/TileBooking.tsx

```diff
@@ -32,7 +32,7 @@ const BookingPill = styled.div<{
     isAnalyticsView && !isExecutingStatus ? '3.5px' : isExecutingStatus ? '10px' : '5px'};
   position: absolute;
   ${({ isAnalyticsView, isExecutingStatus }) =>
-    isAnalyticsView && !isExecutingStatus ? 'right: 1.5rem' : isAnalyticsView ? 'left: -20%' : ''};
+    isAnalyticsView && !isExecutingStatus ? 'right: 1.5rem' : isAnalyticsView ? 'top: 31.5px' : ''};
```

Comment:
@mattydono - does this top margin work correctly under all sizes of the responsive app, given that it is not relative anymore... please check small, large and mobile.

Reply:
Oh and screenshot/gif please

Reply:
I'd suggest using/checking Storybook when making this type of change. It makes testing various states and layouts a breeze (and quickly identifying problems 😉)

Reply:
@rikoe what screen sizes do we support down to?

Reply:
Apple watch! 😂 

Mobile is probably smallest, but the spot tile has a minimum size, so you can only go so small.